src/lib/ares_expand_name.c

```diff
@@ -325,12 +321,13 @@ static ares_ssize_t name_length(const unsigned char *encoded,
 ares_status_t ares__expand_name_for_response(const unsigned char *encoded,
                                              const unsigned char *abuf,
                                              size_t alen, char **s,
-                                             size_t *enclen,
+                                             size_t     *enclen,
                                              ares_bool_t is_hostname)
 {
-  ares_status_t status = ares__expand_name_validated(encoded, abuf, alen, s,
-                                                     enclen, is_hostname);
-  if (status == ARES_EBADNAME)
+  ares_status_t status =
+      ares__expand_name_validated(encoded, abuf, alen, s, enclen, is_hostname);
```

Comment:
This indents *four* spaces all of a sudden. Shouldn't this be indented by two spaces?